.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,11 @@
-:hourglass: **Status**: _Open for visibility_
-:tickets: **Ticket(s)**:
```

Comment:
Can I beg for `## :construction_worker: Changes` here? I like the title at the top (nitpick).

Reply:
Alright, I'll concede the `Changes` header 😛 (to be honest, it seems like a bug that GitHub doesn't strip headers out of PR summaries, so hopefully that gets changed in the future).